x-pack/plugin/deprecation/src/internalClusterTest/resources/plugin-security.policy

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+grant {
+   permission java.lang.RuntimePermission "*", "setContextClassLoader";
```

Comment:
Why is this necessary? I don't see it in the `qa/smoke-test-http` project these tests lived in before?

Reply:
I don't know - this came from a proof-of-concept that @jakelandis did. I removed it and the tests still work.

Reply:
Ah - expect the tests don't appear to shut down correctly without it. Jake, can you comment?

Reply:
sorry, missed this comment. 

This something during the shutdown of Netty. I can remember the exact details, but without this security manager throws an exception that does not have this permission.